torch/onnx/utils.py

```diff
@@ -304,7 +304,8 @@ def _model_to_graph(model, args, verbose=False, training=False,
     if isinstance(model, torch.jit.ScriptModule):
         assert example_outputs is not None, "example_outputs must be provided when exporting a ScriptModule"
         try:
-            method_graph, params = torch._C._jit_pass_lower_graph(model.forward.graph, model._c)
+            model(*args)
```

Comment:
why is this running the model required here?

Reply:
profiled executor requires the model to be run at least once to get the profiled graph.